OWNERS.md

```diff
@@ -0,0 +1,48 @@
+# Rook Maintainers
+
+This page lists all active maintainers and their areas of expertise. This can be used for routing PRs, questions, etc. to the right place.
+
+* See [CONTRIBUTING.md](CONTRIBUTING.md) for general contribution guidelines.
+* See [GOVERNANCE.md](GOVERNANCE.md) for governance guidelines and maintainer responsibilities.
+
+## Senior maintainers
+
+* Bassam Tabbara <bassam@upbound.io> ([bassam](https://github.com/bassam))
+  * Catch-all, architecture, strategy, build and packaging
+* Travis Nielsen <tnielsen@redhat.com> ([travisn](https://github.com/travisn))
+  * Ceph, Operators/Orchestration, Test automation
+* Jared Watts <jared@upbound.io> ([jbw976](https://github.com/jbw976))
+  * Framework, APIs, CockroachDB, Minio
+
+## Maintainers
+
+There are no currently no maintainers.  Maintainers will be added according to the [process for adding a maintainer](GOVERNANCE.md#becoming-a-maintainer) in the governance.
```

Comment:
What about linking `governance` occurences to the `GOVERNANCE.md` like this should work `[governance](/GOVERNANCE.md)` on GitHub.

Reply:
Typo?
> There are NO currently no maintainers.

Reply:
i've incorporated both of these suggestions, thanks @galexrt and @alwinmarkcf!